From dd8dee83813e490a1d7a8715b1c074384688a18c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 22 Nov 2024 14:36:19 +0100
Subject: [PATCH 759/954] feat(stm32mp2): boot BL33 at EL2 by default
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Set BL33 to boot at EL2 by default. The STM32MP_BL33_EL1 must be
manually enabled to boot at EL1.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 7dde583fad68a7aba9617d6a499cdeb65f504e8a feat(stm32mp2): boot BL33 at EL2 by default]
Change-Id: I9c5a183553c5b2f54c55877ecb7b02e04cb333e6
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/420380
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 plat/st/stm32mp2/platform.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/platform.mk b/plat/st/stm32mp2/platform.mk
index bf879d2fb5..9aaa0f4aef 100644
--- a/plat/st/stm32mp2/platform.mk
+++ b/plat/st/stm32mp2/platform.mk
@@ -25,7 +25,7 @@ ifeq ($(ENABLE_PIE),1)
 BL2_IN_XIP_MEM			:=	1
 endif
 
-STM32MP_BL33_EL1		?=	1
+STM32MP_BL33_EL1		?=	0
 ifeq ($(STM32MP_BL33_EL1),1)
 INIT_UNUSED_NS_EL2		:=	1
 endif
-- 
2.53.0

